packages/dapi/scripts/api.js

```diff
@@ -107,17 +107,16 @@ async function main() {
     tenderdashLogger.warn(`Connection closed: ${error.code}`);
   });
 
-  tenderDashWsClient.on('disconnect', () => {
```

Comment:
does it mean we don't handle disconnect event at all anymore? I think we should at least log this.